app/PharBuilder.php

```diff
@@ -455,7 +455,7 @@ protected function addDir($directory, $excludes = array())
              *
              * @var \Symfony\Component\Finder\SplFileInfo $file
              */
-            $this->addFile($directory . DIRECTORY_SEPARATOR . $file->getRelativePathname());
+            $this->addFile(rtrim($directory, '\/') . DIRECTORY_SEPARATOR . $file->getRelativePathname());
```

Comment:
I would prefer that you use the `DIRECTORY_SEPARATOR` constant for this. (And I know, PHP don't care if it's `/` or `\` and even if the two are mixed up, but for the clarity of the code I prefer use a constant when it exist)